tidb-cloud/secure-connections-to-serverless-tier-clusters.md

```diff
@@ -1,50 +1,28 @@
 ---
-title: Secure Connections to Serverless Tier Clusters
-summary: Introduce TLS connection in TiDB Serverless Tier clusters.
+title: TLS connects to Serverless Tier
+summary: Introduce TLS connection in TiDB Serverless Tier.
 ---
 
-# Secure Connections to Serverless Tier Clusters
+# TLS connects to Serverless Tier
 
-This document introduces the core information about TLS in TiDB Serverless Tier.
+Establishing a secure TLS connection between your client and your Serverless database cluster is one of the basic security practices for connecting to your database cluster. The server certificate is issued by an independent third-party certificate provider [Let's Encrypt](https://letsencrypt.org/). You can easily connect to your database cluster without downloading a server-side digital certificate.
 
-## Can I disable TLS in TiDB Serverless Tier?
+## Connection prerequisites
 
-No.
-
-TiDB Serverless Tier allows only TLS connections and rejects non-TLS connections. The reason is that users connect to TiDB Serverless Tier clusters through a public network, so it is really important to use TLS to ensure communication security.
-
-## What TLS versions can I use?
-
-TiDB Serverless Tier supports TLS 1.2 and TLS 1.3. 
-
-TLS 1.0 and TLS 1.1 are not supported due to security reasons. For background information, refer to IETF [Deprecating TLS 1.0 and TLS 1.1](https://datatracker.ietf.org/doc/rfc8996/).
-
-## What certificates do I need？
-
-TiDB Serverless Tier uses certificates from [Let's Encrypt](https://letsencrypt.org/) as a Certificate Authority (CA) for TLS connection between clients and TiDB Serverless Tier clusters. Usually, the root certificate ([ISRG Root X1](https://letsencrypt.org/certs/isrgrootx1.pem.txt)) of Let's Encrypt is present in your system's root CA stores. If the client uses the system's root CA stores by default, such as Java and Go, you can easily connect securely to TiDB Serverless Tier clusters without specifying the path of CA roots.
+Before connecting to your serverless cluster, you can refer to [Password authentication](tidb-cloud-password-authentication.md) or [SSO authentication]() to log in to TiDB Cloud.
```

Comment:
How to call our TiDB Serverless? Serverless Tier, serverless cluster, Serverless cluster, TiDB Serverless Tier, or TiDB Serverless Tier cluster? 
We need to negotiate a common name.

Reply:
TiDB Serverless Tier cluster maybe a good name, please refer to which one is a common name